app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/multi.phtml

```diff
@@ -36,3 +36,12 @@
         <?php endif; ?>
     </div>
 </div>
+<fieldset id="product_composite_configure_fields_qty"
+          class="fieldset product-composite-qty-block admin__fieldset <?= $block->getIsLastFieldset() ? 'last-fieldset' : '' ?>">
+    <div class="field admin__field">
+        <label class="label admin__field-label"><span><?= $block->escapeHtml(__('Quantity')) ?></span></label>
```

Comment:
Please use `$escaper` instead of `$block->escape...`
I would also like you to change all other escaping cases in this template with new approach.